xbmc/dialogs/GUIDialogKeyboardGeneric.h

```diff
@@ -64,6 +66,11 @@ class CGUIDialogKeyboardGeneric : public CGUIDialog, public CGUIKeyboard
     void Backspace();
     void SetEditText(const std::string& text);
     void SendSearchMessage();
+    float getStringWidth(const std::wstring& utf16);
+    void ChangeWordList(int direct);  // direct: 0 - first page, 1 - next page, -1 - prev page
+    void _ChangeWordList(int direct); // direct: 0 - current page, 1 - next page, -1 -prev page
```

Comment:
Please find a better name for this method as there's no way to know the difference when seeing the method names.